reference/tax_provider.yml

```
@@ -848,7 +848,7 @@ components:
           description: A unique identifier for this consignment. Must match the ID of the corresponding Document Request.
         external_id:
           type: string
-          description: 'An optional unique identifier for the document stored in the external providers system. Currently unused in any end to end operation, but may be logged by BigCommerce and hence helpful in issue resolution.'
+          description: 'An optional unique identifier for the document stored in the external providers system. Currently unused in any end-to-end operation, but maybe logged by BigCommerce and hence helpful in issue resolution.'
```

```suggestion
          description: 'An optional unique identifier for the document stored in the external provider's system. Currently not used in any end-to-end operation, but may be logged by BigCommerce and thus be helpful when resolving issues.'
```
this `may be`, not `maybe`.